weasyl/searchtag.py

```diff
@@ -178,6 +178,27 @@ def get_or_create(name):
     return _get_or_create(d.get_search_tag(name))
 
 
+@region.cache_on_arguments()
```

Comment:
This will use the entire list as the cache key – see existing uses of `cache_multi_on_arguments`, which can be folded into the above as

```python
def _get_or_create(*name: str) -> list[str]: ...
```